docs/GettingStarted/QuickStart.md

```diff
@@ -5,6 +5,7 @@
 - [Show snippets using regions](./Regions.md)
 - [Verify your Project](./Verify.md)
 - [Passing Arguments](./PassingArgs.md)
+- [Using Read-only Snippets](./ReadOnlySnippets.md)
```

Comment:
You'll want to make an equivalent change in the other files that also have these nav links.